lib/hsa/hsaTrackingBeamform.hpp

```diff
@@ -58,32 +60,33 @@
  */
 
 
-class hsaBeamformPulsar : public hsaCommand {
+class hsaTrackingBeamform : public hsaCommand {
 public:
     /// Constructor, also initializes internal variables from config and initializes the array of
     /// phases.
-    hsaBeamformPulsar(kotekan::Config& config, const std::string& unique_name,
-                      kotekan::bufferContainer& host_buffers, hsaDeviceInterface& device);
+    hsaTrackingBeamform(kotekan::Config& config, const std::string& unique_name,
+                        kotekan::bufferContainer& host_buffers, hsaDeviceInterface& device);
 
     /// Destructor, cleans up local allocs.
-    virtual ~hsaBeamformPulsar();
+    virtual ~hsaTrackingBeamform();
 
     /// Allocate kernel argument buffer, set kernel dimensions, enqueue kernel
     hsa_signal_t execute(int gpu_frame_id, hsa_signal_t precede_signal) override;
 
 private:
     /// Input length, should be nsamp x n_elem x 2 for complex / 2 since we pack two 4-bit in one
     int32_t input_frame_len;
-    /// Output length, should be 10psr x nsamp x 2 pol x 2 for complex / 2 since we pack two 4-bit
-    /// in one
+    /// Output length, should be 10tracking x nsamp x 2 pol x 2 for complex / 2 since we pack two
```

Comment:
```suggestion
    /// Output length, should be num_beams x nsamp x 2 pol x 2 for complex / 2 since we pack two
```
I think that should be `num_beams` since this is no longer fixed at 10